packages/dev/loaders/test/unit/Interactivity/flow nodes.test.ts

```diff
@@ -0,0 +1,1752 @@
+import { NullEngine } from "core/Engines";
```

Comment:
Comment to self - on top of the implemented unit tests, add vis-tests for the FlowGraph itself.

Reply:
1 vis-test was added, on top of all of the unit tests already available in this PR.